src/ParallelRun.jl

```diff
@@ -322,60 +322,24 @@ function infer_init(rcfs::Vector{RunCamcolField},
     return catalog, target_sources
 end
 
-"""
-A simplified helper method to choose between one_node_single_infer and one_node_joint_infer
-"""
-function parallel_infer(catalog, target_sources, neighbor_map, images;
-                        joint_infer=false,
-                        joint_infer_n_iters=50,
-                        joint_infer_batch_size=60,
-                        joint_infer_shuffle=true,
-                        reserve_thread=Ref(false),
-                        thread_fun=phalse,
-                        timing=InferTiming())
```

Comment:
No longer necessary because we just pass the particular inference method to `one_node_infer`, rather than passing it a flag. @agnusmaximus let me know if this doesn't make sense.